keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -29,9 +29,25 @@ abstract class DeliveryArtifact {
   abstract val name: String
   @Discriminator abstract val type: ArtifactType
   abstract val versioningStrategy: VersioningStrategy
-  abstract val reference: String // friendly reference to use within a delivery config
-  abstract val deliveryConfigName: String? // the delivery config this artifact is a part of
```

Comment:
These comments would be better as kdoc comments and not inline

Reply:
Uhh, that's exactly what I did on the right side of the diff? 🤔 😄

Reply:
OH I mean at the top of the method like with 
/**
* @param reference 
*/

Reply:
wow formatting there did not work

Just a style preference. But I prefer docs outside the method if there is more than one or two comments

Reply:
But this is not a method, it's the body of the class...

Reply:
🤦  everything is great over here Luis. I am doing great. I'm going to blame this one on wishing github would show me a couple more surrounding lines (but obviously only when I want them) (but also without clicking that, cause how could I know I wanted them)